src/main/java/core/basesyntax/Figures.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public abstract class Figures implements Area, Drawable {
```

Comment:
Class names should be in singular form
```suggestion
public abstract class Figure implements Area, Drawable {
```